pkg/config/templates/rancherd-15-logging.yaml

```diff
@@ -0,0 +1,70 @@
+resources:
+- apiVersion: management.cattle.io/v3
+  kind: ManagedChart
+  metadata:
+    name: rancher-logging
+    namespace: fleet-local
+  spec:
+    chart: rancher-logging
+    releaseName: rancher-logging
+    version: {{ .LoggingChartVersion }}
+    defaultNamespace: cattle-logging-system
+    repoName: harvester-charts
+    targets:
+    - clusterName: local
+      clusterSelector:
+        matchExpressions:
+        - key: provisioning.cattle.io/unmanaged-system-agent
+          operator: DoesNotExist
+    values:
+      disablePvc: true
+      additionalLoggingSources:
+        kubeAudit:
+          auditFilename: ""
+          enabled: false
+          pathPrefix: ""
+          fluentbit:
+            logTag: kube-audit
+            tolerations:
+            - key: node-role.kubernetes.io/controlplane
+              value: "true"
+              effect: NoSchedule
+            - key: node-role.kubernetes.io/etcd
+              value: "true"
+              effect: NoExecute
+      fluentd:
+        resources:
+          limits:
+            cpu: 1000m
+            memory: 800Mi
+          requests:
+            cpu: 100m
+            memory: 200Mi
+      fluentbit:
+        resources:
+          limits:
+            cpu: 200m
+            memory: 200Mi
+          requests:
+            cpu: 50m
+            memory: 50Mi
+- apiVersion: logging.banzaicloud.io/v1beta1
+  kind: ClusterFlow
+  metadata:
+    name: harvester-loki
+    namespace: cattle-logging-system
+  spec:
+    match:
+      - select: { }
+    globalOutputRefs:
+      - harvester-loki
+- apiVersion: logging.banzaicloud.io/v1beta1
+  kind: ClusterOutput
+  metadata:
+    name: harvester-loki
+    namespace: cattle-logging-system
+  spec:
+    loki:
+      url: http://loki-stack.cattle-logging-system.svc:3100
+      extra_labels:
+        logOutput: harvester-loki
```

Comment:
From user's perspective it will be nice to have some built-in logging dashboard, but adding loki as the default cluster flow and output has brought some additional concerns with support and maintenance: cc @bk201
1. are those logs been persistend, and will the exising volume size enough to store those data or log expantion.
2. does Loki or the cattle-logging-system contians any resource quota and limitations, as at times, logging aggregation and filtering can really drain the node resources.
3. are we ready/plan to support it if we are setting it as the default system integration.

Reply:
Per discussion, we are going to have a guide to setup Loki on day 2 when needed.